### ISSUE TITLE
Deleting/unmounting files

### ISSUE THREAD

**Issue by nictru** [biowasm/aioli#39]
RAM usage becomes a problem for my project after processing some files. I would appreciate if I had an option to delete files that are not needed any more. How do folks usually handle this?

**Comment by robertaboukhalil** [comment]
Hi @nictru, usually people will do this by calling `CLI.fs.unlink("/some/path/here")` after not needing a file anymore. If that doesn't work, I'm curious to hear more about your use case, and also which kinds of files you're mounting? e.g. Files the user selects from their computer, strings mounted as files, or URLs that are mounted to the file system?

**Comment by nictru** [comment]
Hey @robertaboukhalil, 
thanks for your great work and your quick response. Tested the approach and it worked just fine. 

When I was looking for a potential solution, I was mostly relying on the [documentation](https://biowasm.com/documentation#aioli). I also went through the source code in [this file](https://github.com/biowasm/aioli/blob/main/src/worker.js) and although I now saw that the `unlink` function is used once in that file, it was not clear to me how I can use the `fs` object in my project.

So maybe it would be helpful for other users to add this solution to the documentation?
